pkg/aws/instance_info.go

```diff
@@ -35,6 +35,10 @@ func (i Instance) AvailableStorage(instanceStorageScaleFactor float64, rootVolum
 	return int64(instanceStorageScaleFactor * float64(i.InstanceStorageDevices*i.InstanceStorageDeviceSize))
 }
 
+func (i Instance) MemoryFraction(fraction int64) int64 {
+	return int64(float64(fraction) / 100.0 * float64(i.Memory))
```

Comment:
Since input and output are both `int`, there's a nice way to avoid the floats:
```go
func (i Instance) MemoryFraction(percent int64) int64 {
	return percent * i.Memory / 100
}
```

Reply:
Thanks! Changed!